cle/loader.py

```diff
@@ -218,6 +219,12 @@ def memory(self) -> Clemory:
             raise ValueError("Cannot access memory before loading is complete")
         return result
 
+    @property
+    def memory_ro_view(self) -> ClemoryReadOnlyView | None:
+        if self._memory is None:
+            raise ValueError("Cannot access memory_ro_view before loading is complete")
+        return self._memory_ro_view
```

Comment:
this is gonna be a type error - should be `if self._memory_ro_view is not None`

Reply:
It’s actually intended - I’ll add a comment to explain.